scripts/travis_install.sh

```diff
@@ -41,8 +41,8 @@ case "$TASK" in
                     "2.7")
                         TENSORFLOW_PACKAGE_URL="https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp27-none-linux_x86_64.whl"
                     ;;
-                    "3.4")
-                        TENSORFLOW_PACKAGE_URL="https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp34-cp34m-linux_x86_64.whl"
+                    "3.5")
+                        TENSORFLOW_PACKAGE_URL="https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp35-cp35m-linux_x86_64.whl"
```

Comment:
@cclauss This is causing a Missing wheel error at workflow build

Reply:
https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp35-cp35m-linux_x86_64.whl does not exist